tests/test_cli/test_test.py

```diff
@@ -337,3 +337,30 @@ def test_run(
                 str(package_dirpath),
             )
             assert result.exit_code == OK_PYTEST_EXIT_CODE
+
+
+@_parametrize_class
+class TestPackageTestByPathWithCov(BaseAEATestCommand):
+    """Test that the command 'aea test by-path path-to-package' works as expected (non-empty test suite)."""
+
+    @pytest.mark.parametrize("package_type", list(ComponentType))
```

Comment:
don't need to call `list` on an `Enum`, it is already an iterable. And you know this as well, since you call list on it, which wouldn't work if it wasn't an iterable to begin with.